rs/boundary_node/rate_limits/api/src/schema_versions/v1.rs

```diff
@@ -0,0 +1,69 @@
+use candid::Principal;
+use regex::Regex;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+const DOUBLE_INDENT: &str = "      ";
+
+// Defines the rate-limit rule to be stored in the canister
+#[derive(Serialize, Deserialize, Debug)]
+pub struct RateLimitRule {
```

Comment:
The request type is missing. Besides the limit, everything can be optional as far as I remember.

Reply:
Added `request_type` field and also changed `pub methods: Regex` to `pub methods_regex: Option<String>`. Having `Option<Regex>` makes things a bit complex. I can change it later, separately.